### PR TITLE
testing/checkpolicy: initial import

### PR DESCRIPTION
**Pull request by tych0**
Signed-off-by: Tycho Andersen <tycho@docker.com>

### CODE REVIEW

File: testing/checkpolicy/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+install=""
+subpackages="$pkgname-doc"
+source="https://github.com/SELinuxProject/selinux/archive/checkpolicy-$pkgver.tar.gz
+	0001-remove-apple-ifdef.patch
+"
+builddir="$srcdir/selinux-checkpolicy-$pkgver/checkpolicy"
+
+build() {
+	cd "$builddir"
+  make || return 1
```
Comment (jirutka) [inline testing/checkpolicy/APKBUILD]: Please use only tabs for indentation (and alignment) in APKBUILDs, not spaces.

File: testing/checkpolicy/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+pkgname=checkpolicy
+pkgver=2.6
+pkgrel=0
+pkgdesc="SELinux policy checker"
+url="http://www.selinuxproject.org"
+arch="all"
+license="GPLv2"
+depends="flex-dev"
+depends_dev=""
+makedepends="$depends_dev libsepol-dev bison flex flex-dev"
```
Comment (jirutka) [inline testing/checkpolicy/APKBUILD]: This package doesn’t have -dev subpackage, so `$depends_dev` is useless here. Please remove both declaration of `depends_dev` variable and its usage in `makedepends`.

File: testing/checkpolicy/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Maintainer: Tycho Andersen <tycho@docker.com>
+pkgname=checkpolicy
+pkgver=2.6
+pkgrel=0
+pkgdesc="SELinux policy checker"
+url="http://www.selinuxproject.org"
+arch="all"
+license="GPLv2"
+depends="flex-dev"
```
Comment (jirutka) [inline testing/checkpolicy/APKBUILD]: This looks suspicious. Packages usually don’t depend on -dev (i.e. .h .a, .la, … files). Are you sure that checkpolicy needs flex-dev, not just flex?

Comment (tych0) [inline testing/checkpolicy/APKBUILD]: Well, it seems not:

```
alpine-edge:~/aports/testing/checkpolicy/pkg/checkpolicy/usr/bin$ ldd checkmodule 
	/lib/ld-musl-x86_64.so.1 (0x55736fbc8000)
	libc.musl-x86_64.so.1 => /lib/ld-musl-x86_64.so.1 (0x55736fbc8000)
alpine-edge:~/aports/testing/checkpolicy/pkg/checkpolicy/usr/bin$ ldd checkpolicy 
	/lib/ld-musl-x86_64.so.1 (0x5624657fd000)
	libc.musl-x86_64.so.1 => /lib/ld-musl-x86_64.so.1 (0x5624657fd000)
```

However, the build system passes -lfl and libfl.so lives in flex-dev, so I just assumed it did. Anyway, I took it out and addressed the rest of your comments. Thanks!

**Pull request closed without merging**